packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -63,8 +72,14 @@ export class MarketOperationUtils {
             ...opts,
         };
         const [makerToken, takerToken] = getOrderTokens(nativeOrders[0]);
-        const [fillableAmounts, dexQuotes] = await this._sampler.executeAsync(
+        const [fillableAmounts, ethToMakerAssetRate, dexQuotes] = await this._sampler.executeAsync(
             DexOrderSampler.ops.getOrderFillableTakerAmounts(nativeOrders),
+            DexOrderSampler.ops.getMedianSellRate(
+                difference(FEE_QUOTE_SOURCES, _opts.excludedSources),
+                this._wethAddress,
+                makerToken,
+                ONE_ETHER,
+            ),
```

Comment:
Right now this does not use native orders in the interest of response time (don't want an additional query to mesh). It probably doesn't really matter though since we really just need a reasonable estimate.